components/shared_code/src/shared_data_model/logos/__init__.py

```diff
@@ -1,5 +1,14 @@
-"""Define this as the path to the logos."""
+"""Logos for sources."""
 
-import pathlib
+import importlib.resources
 
-LOGOS_ROOT = pathlib.Path(__file__).parent.absolute()
+
+def get_logo(name: str) -> bytes:
+    """Return the logo."""
+    return importlib.resources.read_binary("shared_data_model.logos", f"{name}.png")
+
+
+def get_logo_names() -> list[str]:
```

Comment:
Perhaps this should be memoized in some const like `LOGO_NAMES`?

Reply:
It's only used once per run, when the data model is instantiated.